application/common/config/main.php

```diff
@@ -136,16 +135,16 @@
                     'logVars' => [], // Disable logging of _SERVER, _POST, etc.
                     'message' => [
                         'to' => $alertsEmail ?? '(disabled)',
-                        'subject' => 'ALERT - ' . $idpName . ' PW [env=' . $appEnv . ']',
+                        'subject' => 'ALERT - ' . $idpName . ' PW [env=' . YII_ENV . ']',
                     ],
                     'baseUrl' => $emailServiceConfig['baseUrl'],
                     'accessToken' => $emailServiceConfig['accessToken'],
                     'assertValidIp' => $emailServiceConfig['assertValidIp'],
                     'validIpRanges' => $emailServiceConfig['validIpRanges'],
                     'enabled' => $alertsEmailEnabled,
-                    'prefix' => function($message) use ($appEnv) {
+                    'prefix' => function($message) use (YII_ENV) {
```

Comment:
I don't think you need this in the `use` parens, since it's a global constant:
```suggestion
                    'prefix' => function($message) {
```

I think that's why the build failed:
> PHP Parse error:  syntax error, unexpected identifier "YII_ENV", expecting variable or "&" or amp in /data/common/config/main.php on line 145

Reply:
I did that correct in id-broker, and I saw the red squiggle under the filename. But when I hovered over the scrollbar it showed a warning that I didn't need to address (Unused parameter "message"). Should have looked a little closer.